prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MO/MO-receiver-transform.yml

```diff
@@ -0,0 +1,40 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+elements:
+  - name: mo-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
+
+  - name: mo-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
+
+  - name: mo-sender-facility-namespace-Id
+    value: [ '"MOELR"' ]
+    hl7Spec: [ 'MSH-5-1' ]
+
+  - name: mo-sender-facility-universal-Id
+    value: [ '"2.16.840.1.114222.4.3.3.36"' ]
+    hl7Spec: [ 'MSH-5-2' ]
+
+  - name: mo-sender-facility-universal-id-type
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-5-3' ]
+
+  - name: mo-receiving-application-namespace-Id
+    value: [ '"MODHSS"' ]
+    hl7Spec: [ 'MSH-6-1' ]
+
+  - name: mo-receiving-application-universal-Id
+    value: [ '"2.16.840.1.114222.4.3.2.2.1.163"' ]
+    hl7Spec: [ 'MSH-6-2' ]
+
+  - name: mo-receiving-application-universal-id-type
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-6-3' ]
```

Comment:
According to the setting above (suppressHl7Fields: "MSH-19-3, PID-15-3, ORC-14-1, OBR-17-1"), you can suppress them here as following:

  - name: mo-unwanted-fields
    condition: 'true'
    value: [ '""' ]
    hl7Spec: [ 'MSH-19-3', '%{PID}-13-12', '%{PID}-15-3', '%{ORC}-14-1', '%{OBR}-17-1'  ]